content/engine/install/rhel.md

```diff
@@ -185,27 +182,43 @@ download the `.rpm` file for your release and install it manually. You need to
 download a new file each time you want to upgrade Docker Engine.
 
 <!-- markdownlint-disable-next-line -->
-1. Go to [{{% param "download-url-base" %}}/]({{% param "download-url-base" %}}/)
-   and choose your version of RHEL. Then go to `s390x/stable/Packages/`
-   and download the `.rpm` file for the Docker version you want to install.
+1. Go to [{{% param "download-url-base" %}}/]({{% param "download-url-base" %}}/).
+
+2. Select your RHEL version in the list.
+
+3. Go to `pool/stable/` and select the applicable architecture (`x86_64`,
```

Comment:
I don't see `pool/stable/` for versions I've peeked at.  After choosing RHEL version you choose the architecture, then stable, then Packages.
for example: `https://download.docker.com/linux/rhel/9/x86_64/stable/Packages/`

Reply:
Sorry, copy-paste error